deployment/SharedCustomerReources.template.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
You'll want to sync with Jeff about when and why the customer would use this template.